lib/src/core.rs

```diff
@@ -10,9 +11,11 @@ pub enum ProcessResult {
     Done(NodeData),
 }
 
-#[derive(Clone)]
+#[derive(Clone, Getters, Debug)]
 pub struct Core {
+    #[get = "pub"]
     pub is_idle: bool,
+    #[get = "pub"]
```

Comment:
```suggestion
    #[get_copy = "pub with_prefix"]
```

Reply:
Changed.

Reply:
If you need to copy, implement a method to return clone.